patches/@keystonehq+bc-ur-registry+0.4.4.patch

```diff
@@ -0,0 +1,1830 @@
+diff --git a/node_modules/@keystonehq/bc-ur-registry/src/Bytes.ts b/node_modules/@keystonehq/bc-ur-registry/src/Bytes.ts
```

Comment:
The `@keystonehq/bc-ur-registry` package contains TypeScript files in its distributed form. This is not necessary or recommended for TypeScript-compatible libraries (rather, it is enough to publish type _definition_ files). The issue for this is logged here: https://github.com/KeystoneHQ/ur-registry/issues/13. In the meantime, as our code works fine presently without any TypeScript files, we do not need these in order to use this library. This makes the new `tsc` step in `yarn lint` pass.

Reply:
So, to make sure I understand correctly, this code is not used by the extension, but we need this patch to get lint passing?

Reply:
Sorry maybe my explanation was confusing. We do use this package (it looks like a [component that is involved with the QR code stuff](https://github.com/MetaMask/metamask-extension/blob/b8fc1ae671c153d84de6caa254de3e55c153de75/ui/components/app/qr-hardware-popover/qr-hardware-sign-request/reader.js#L2) imports `@keystonehq/bc-ur-registry-eth`, which imports `@keystonehq/bc-ur-registry`). However, the way this package has been bundled and published to NPM is incorrect as it includes TypeScript files.